.github/workflows/integration.yml

```
@@ -162,13 +162,25 @@ jobs:
     runs-on: ubuntu-24.04
     timeout-minutes: 20
     strategy:
+      # Test latest upstream Gateway API versions against the latest Kubernetes
+      # version, and test the minimum Kubernetes version against the (vendored)
+      # minimum Gateway API version.
       matrix:
         k8s:
-          - v1.23
           - v1.32
-        linkerd-managed-gw-crds:
-          - true
-          - false
+        gateway-api:
+          - version: v1.2.1
```

Actually, we probably want to test the MSGV here as well (with only the standard subset).

```suggestion
          - version: v1.1.1
```

because we already cover the latest experimental featureset.